content/applications/inventory_and_mrp/manufacturing/management/unbuild_orders.rst

```diff
@@ -0,0 +1,90 @@
+==============
+Unbuild orders
+==============
+
+.. |BoM| replace:: :abbr:`BoM (Bill of Materials)`
+
+In some cases, it is necessary to dismantle manufactured products into their individual components.
+This may be required if too many units of a product were built, or if the components of one product
+must be reclaimed to use in the manufacturing of another.
+
+In Odoo *Manufacturing*, products can be dismantled, and their components returned to inventory,
+using *unbuild orders*. By using unbuild orders to accomplish this task, inventory counts for the
+finished product and its components remain accurate, based on the quantity of products dismantled,
+and the quantity of components reclaimed.
+
+Create unbuild order
+====================
+
+A new unbuild order can be created by navigating to :menuselection:`Manufacturing app --> Operations
+--> Unbuild Orders`, and clicking :guilabel:`New`.
+
+Begin filling out the new unbuild order by selecting a :guilabel:`Product` to unbuild. After doing
+so, the :guilabel:`Bill of Material` field auto-populates with the corresponding bill of materials
+(BoM). If a different |BoM| should be used, click on the :guilabel:`Bill of Material` field, and
+select it from the drop-down menu.
+
+Alternatively, a specific |BoM| can be selected in the :guilabel:`Bill of Material` field before
+selecting a product, which causes the corresponding product to auto-populate in the
+:guilabel:`Product` field.
+
+Next, specify the :guilabel:`Quantity` of the product that is being unbuilt.
+
+If the product being unbuilt was originally manufactured in a specific manufacturing order (MO),
+select it in the :guilabel:`Manufacturing Order` field.
+
+In the :guilabel:`Source Location` field, select the location where the product being unbuilt is
+currently stored.
+
+In the :guilabel:`Destination Location` field, select the location where the reclaimed components
+are stored after the unbuild order is completed.
+
+If the :guilabel:`Lots & Serial Numbers` feature is enabled in the settings of the *Inventory* app,
+a :guilabel:`Lot/Serial Number` field appears on the unbuild order, which can be used to specify the
```

Comment:
The text for the "Lots & Serial Numbers" feature in the settings is not present on the UI in this instruction, so no guilabel :)
```suggestion
If the *Lots & Serial Numbers* feature is enabled in the settings of the *Inventory* app, a
:guilabel:`Lot/Serial Number` field appears on the unbuild order, which can be used to specify the
```